src/GuidesIndex.php

```diff
@@ -25,6 +25,24 @@ private static function getSummaries(
     return require(BuildPaths::GUIDES_SUMMARY);
   }
 
+  public static function search(string $term, SearchResultSet &$results): void {
```

Comment:
yeah, objects are byref (in PHP too), so the `&` is unneeded - still, the other suggested approaches here would be cleaner than depending on that.